### PR TITLE
Add ticks for type variable pretty-printing in BbML

### PR DESCRIPTION
**Pull request by NeilKleistGao**

### CODE REVIEW

**Comment by LPTK** [comment]
I think skolems can be shown as simply `A` (like, again, in the legacy compiler) instead of the very strange-looking `'<A>`.

File: hkmc2/shared/src/test/mlscript/bbml/bbCheck.mls [hunk 1]
```diff
@@ -92,11 +92,11 @@ high(x => x + 1)
 //│ ╔══[ERROR] Type error in reference with expected type Int
 //│ ║  l.91: 	high(x => x + 1)
 //│ ║        	          ^
-//│ ╙── because: cannot constrain  '<A>  <:  Int
-//│ ╔══[ERROR] Type error in application with expected type '<A>
+//│ ╙── because: cannot constrain  'A  <:  Int
```
Comment (LPTK) [inline hkmc2/shared/src/test/mlscript/bbml/bbCheck.mls]: It's not right. It has a tick...

Comment (NeilKleistGao) [reply hkmc2/shared/src/test/mlscript/bbml/bbCheck.mls]: Previously I was worried that using the same name for skolems and variables is confusing. But I just realized it is actually fine. 😅

**Review by LPTK — APPROVED** [review]
LGTM

**Pull request merged**